pkg/unikontainers/unikernels/unikraft.go

```diff
@@ -83,6 +85,37 @@ func (u *Unikraft) Init(data UnikernelParams) error {
 	return nil
 }
 
+func (u *Unikraft) configureNet(ethDeviceIP, ethDeviceGateway, ethDeviceMask string) {
+	setCompatNetConfig := func() {
+		u.Net.Address = "netdev.ipv4_addr=" + ethDeviceIP
+		u.Net.Gateway = "netdev.ipv4_gw_addr=" + ethDeviceGateway
+		u.Net.Mask = "netdev.ipv4_subnet_mask=" + ethDeviceMask
+	}
+
+	setCurrentNetConfig := func() {
+		u.Net.Address = "netdev.ip=" + ethDeviceIP + "/24:" + ethDeviceGateway + ":8.8.8.8"
+	}
+
+	if u.Version == "" {
+		setCurrentNetConfig()
+		return
+	}
+
+	unikernelVersion, err := version.NewVersion(u.Version)
+	if err != nil {
+		setCurrentNetConfig()
```

Comment:
Maybe we can add an error message here and log the use of a specific cli option (e.g. new Unikraft version)

Reply:
Since there is no logger available inside the `unikontainers/unikernels` package, our best option is to return an error and have the caller check against this specific error and log it.